### PR TITLE
docs: Add SECURITY.md with instructions on reporting vulnerabilities

### PR DESCRIPTION
**Pull request by MthwRobinson**
### Summary

Addresses [CORE-149](https://unstructured-ai.atlassian.net/browse/CORE-149). Adds a `SECURITY.md` file with instructions on how to report security vulnerabilities. Since this is in the `.github` repo, this will be applied as the default security policy for all repos in the organization. The format is based on the GitHub's suggested format.

### CODE REVIEW

**Comment by qued** [comment]
I can't confirm that the security policy shows up when someone opens an issue in one of our repos, but presumably that would only happen once this is merged.

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+| Package | Version | Supported          |
+| ------- | ------- | ------------------ |
+| `unstructured` | All versions   | :white_check_mark: |
+| `unstructured-api-tools` | All versions   | :white_check_mark: |
+| `pipeline-sec-filings` | All versions   | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+If you find a security vulenerability in one of our projects, email us
```
Comment (qued) [inline SECURITY.md:15]: Should we emphasize that they _not_ open an issue?

**Review by qued — APPROVED** [review]
LGTM other than comment.

File: SECURITY.md (hunk 1 above)
Comment (MthwRobinson) [reply SECURITY.md:15]: Added that in there

**Pull request merged**